src/main/Extensions/VSCode/VSCodeExtension.ts

```diff
@@ -9,6 +9,7 @@ import { searchFilter } from "@common/Core/Search/SearchFilter";
 import Database from "better-sqlite3";
 import * as Path from "path";
 import * as Url from "url";
+import type { Logger } from "../../Core/Logger";
```

Comment:
```suggestion
import type { Logger } from "@Core/Logger";
```

Reply:
Is there any way to get vscode to automatically pick these instead?

Reply:
I don't know, I always get the suggestions.

Reply:
Oh interesting i always get relative paths which is annoying must be some setting or extension related thing![image](https://github.com/user-attachments/assets/51c91d10-d8b8-444f-a6ad-580c1a62bb01)

Reply:
Ah found it had to set this "typescript.preferences.importModuleSpecifier": "non-relative",